src/denovo_discovery/denovo_utils.cpp

```diff
@@ -86,26 +86,26 @@ PathComponents find_interval_and_flanks_in_localpath(const Interval &interval,
 }
 
 
-std::set<MinimizerHitPtr, pComp_path>
-find_hits_inside_path(const std::set<MinimizerHitPtr, pComp_path> &read_hits, const prg::Path &local_path) {
-    std::set<MinimizerHitPtr, pComp_path> hits_inside_local_path;
+std::vector<MinimizerHitPtr>
+find_hits_inside_path(const std::vector<MinimizerHitPtr> &read_hits, const prg::Path &local_path) {
+    std::vector<MinimizerHitPtr> hits_inside_local_path;
 
     if (local_path.empty()) {
         return hits_inside_local_path;
     }
 
     for (const auto &current_read_hit : read_hits) {
         for (const auto &interval : local_path) {
-            const auto hit_is_to_left_of_path_start { interval.start > current_read_hit->prg_path.get_end() };
+            const auto hit_is_to_left_of_path_start { interval.start > current_read_hit->get_prg_path().get_end() };
```

Comment:
We now seem to make 3 calls to `current_read_hit->get_prg_path()`. Would it better to create a reference at the beginning of each loop iteration?

Reply:
Done in https://github.com/leoisl/pandora/commit/464e00502d573fdd684cde18e7ea1947bc980cec